### PR TITLE
Fix data import on linux alpine

### PR DESCRIPTION
**Pull request by akcyp**
### What does it do?

Change fs-extra `rename` function (which is native `fs` function, see https://github.com/jprichardson/node-fs-extra/blob/master/lib/fs/index.js#L33) to fs-extra `move` method that works on multiple plaforms https://github.com/jprichardson/node-fs-extra/blob/master/docs/move.md

### Why is it needed?

This PR fixes bug: https://github.com/strapi/strapi/issues/15724

### How to test it?

Test `strapi import` on multiple platforms.

### Related issue(s)/PR(s)

[Let us know if this is related to any issue/pull request](https://github.com/strapi/strapi/issues/15724)

### CODE REVIEW

**Review by christiancp100 — APPROVED** [review]
TL;DR: LGTM 🍝


The right terminal is showing the current failure and the left terminal is the same docker image (alpine with node 14, attached below) running the code in this fix branch. As you can see it works.
Screenshot 2023-02-08 at 13 30 53

The next step was checking that this fix doesn't break the current export/import flow in other Operating Systems: 
Screenshot 2023-02-08 at 14 10 21

Finally, manually throw an error and check if it still does the rollback. If the new added logic didn't work, the error thrown would be:
There was an error doing the rollback process. The original files are in ${backupDirectory}, but we failed to restore them to ${assetsDirectory}, but instead, the error is:
Screenshot 2023-02-08 at 14 18 38

```
FROM node:14-alpine

RUN apk update && \
    apk add --no-cache git python3 py3-pip make g++


WORKDIR /app

RUN git clone https://github.com/strapi/strapi.git
# or RUN git clone https://github.com/akcyp/strapi.git


WORKDIR /app/strapi
# and RUN git checkout alpine-import-fix

RUN yarn setup

CMD [ "sh" ]
```

**Comment by exotexot** [comment]
Sadly I can't confirm that this is working for me.

 "@strapi/strapi": "4.6.1"
FROM node:16-alpine

**Comment by christiancp100** [comment]
> 

Hey @exotexot! Could you send us more information to be able to reproduce it? Since the error is different you might have a small difference in your setup (Dockerfile, Kubernetes files, etc...) that could help us identify the error.

**Comment by exotexot** [comment]
@christiancp100 

Hey! I'm using https://github.com/strapi-community/strapi-tool-dockerize to create my docker environment for strapi.

It generates this Dockerfile:

```
FROM node:16-alpine
# Installing libvips-dev for sharp Compatability
RUN apk update && apk add --no-cache build-base gcc autoconf automake zlib-dev libpng-dev nasm bash vips-dev
ARG NODE_ENV=development
ENV NODE_ENV=${NODE_ENV}
WORKDIR /opt/
COPY ./package.json ./package-lock.json ./
ENV PATH /opt/node_modules/.bin:$PATH
RUN npm install
WORKDIR /opt/app
COPY ./ .
RUN npm run build
EXPOSE 1337
CMD ["npm", "run", "develop"]
``
